libs.glassfish_logging/external/commons-logging-1.0.4-license.txt

```diff
@@ -0,0 +1,206 @@
+Name: Commons Logging
+License: Apache-2.0
+Description: Commons Logging
+Version: 1.0.4
```

Comment:
Please add a `Origin` header - it is required. I would let it be: `Distributed with Sun GlassFish Enterprise Server v2.1`. You can run license verification by invoking: `ant verify-all-libs-and-licenses` and look for the module you are checking.